tests/bench.py

```diff
@@ -5,7 +5,7 @@
 import numpy as np
 import pytest
 
-import geff.networkx as geff_nx
+import geff.networkx.io as geff_nx
```

Comment:
It's a little weird to do this instead of just having `import geff` here and `geff.write_nx` below (since we are now recommending that usage pattern)

Reply:
Same comment for the other tests. I'd prefer at least one of them use the new import structure (for both read_nx/write_nx and validate) to make sure it worked.

Reply:
Yup fair point. Pretty sure I just did whatever required the smallest change at the time. Both test files are now update to use a direct import geff.